tools/install/windows/merge/BrokerMerge.wxs

```diff
@@ -17,7 +18,7 @@
       <Directory Id="$(var.ProgramFilesFolder)">
         <Directory Id="PROGRAM_FILES_ETC_DIR" Name="ETC">
           <Directory Id="INSTALLDIR" Name="RDMnetBroker" FileSource="$(var.InstallSourceDir)">
-            <Component Id="Broker" Guid="2B4014E4-5278-469A-ABA7-A94F0AB850DA" SharedDllRefCount='yes'>
+            <Component Id="Broker" Guid="2B4014E4-5278-469A-ABA7-A94F0AB850DA" SharedDllRefCount="yes">
```

Comment:
Is this explicit mention of the sharedDllRefCount required in addition to the Service elements for the shareddllrefcounts to work correctly?

Reply:
I don't know the answer to that; I was under the impression (I think you told me) that you needed that attribute in order for the component to be properly refcounted.

Reply:
For a normal file, yes that's required. I was just wondering if the service control element required that to function or if the service elements would handle the refcounting on their own.

Reply:
Well I just combined the ServiceControl and File elements under the same component, for convenience, so I haven't tested the alternative.